.github/workflows/tests.yml

```diff
@@ -87,21 +87,12 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ["3.8", "3.9", "3.10", "pypy3.9"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13", "pypy3.9"]
         include:
           - os: ubuntu-latest
             python-version: "3.10"
             with-coverage: true
 
-          - os: ubuntu-latest
-            python-version: 3.11-dev
-            tox-env: devel
-          - os: windows-latest
-            python-version: 3.11-dev
-            tox-env: devel
-          - os: macos-latest
-            python-version: 3.11-dev
-            tox-env: devel
```

Comment:
Why were these removed?

Reply:
It's included in the new matrice with 3.11 (or 3.14-dev when it's going to be available), right ?

Reply:
3.11 yes.

But `devel` is a special target for tox where we run the suite agains pytest main-branch: https://github.com/pytest-dev/pytest-html/blob/master/tox.ini#L32

Reply:
So set it to `3.13`.